.yaspeller.json

```diff
@@ -123,6 +125,7 @@
     "takeaways",
     "Talha",
     "talha",
+    "Talha",
```

Comment:
Talha is already present in the list. Just one row above.

Reply:
Fixed.